src/api/app/models/event_subscription.rb

```diff
@@ -60,11 +60,10 @@ class EventSubscription < ApplicationRecord
 
   after_save :measure_changes
 
-  def self.receiver_roles_to_display(user)
+  # TODO: unify both lists after renaming and migrating the *watchers methods
```

Comment:
Beside this method, both `RECEIVER_ROLE_TEXTS` and `BETA_RECEIVER_ROLE_TEXTS` are used only in this partial:

https://github.com/openSUSE/open-build-service/blob/1c240a0101f63e08c63784949b233736e5a6fac6/src/api/app/views/webui/subscriptions/_subscriptions_form.html.haml#L10

There's nothing preventing us from already merging `BETA_RECEIVER_ROLE_TEXTS` into `RECEIVER_ROLE_TEXTS`. This would simplify the view linked in this comment while also the method `self.receiver_roles_to_display` here.

What do you think?

Reply:
I finally merged them. Please have another look, @dmarcoux